core/__snapshot__/com.karumi.kotlinsnapshot.core.TestClassNameAsDirectoryTest/should create test class name folder with the method name snap file.snap

```diff
@@ -1 +1 @@
-{"name":"gabriel","id":5}
\ No newline at end of file
+"{\"name\":\"gabriel\",\"id\":5}"
```

Comment:
This means if I have a simple `String` would convert to JSON with ugly `\"`?  

I think if we have a String you want to serialize a String. It's like serializing a number.

I don't like this idea 👎 I use this library most of the time for serializing `body` String with json inside it.

Reply:
But this is the only way to use an homogeneous format for all cases. If you want to remove the escaping characters we can still use a custom serializer for these scenarios letting the user decide which one to use depending on their needs

Reply:
Then my question is if a user creates a PR creating `JsonSerializationModuleExceptStrings` should we accept the PR?

Reply:
Unless we see a lot of people requesting it, I wouldn't include it, that's why we made the library extensible so that people can serialize their data with the format they want.

Reply:
I agree with both of you. I've changed the serializer to don't format as a json string a raw not nested string so if our users decide to save a string as is they'll be able to see the same format in the file. However, strings inside nested objects will be formatted as json strings. I think this solution have the same of both worlds.

Reply:
yeah! thanks!